Documentation/devel/stage1-implementors-guide.md

```diff
@@ -38,6 +38,10 @@ In the bundled rkt stage 1 which includes systemd-nspawn and systemd, the entryp
 
 An alternative stage 1 could forego systemd-nspawn and systemd altogether, or retain these and introduce something like novm or qemu-kvm for greater isolation by first starting a VM.  All that is required is an executable at the place indicated by the "coreos.com/rkt/stage1/run" entrypoint which knows how to apply the Pod Manifest and prepared ACI file-systems to good effect.
 
+The resolved entrypoint must inform rkt of its pid for the benefit of `rkt enter`. rkt supports two ways of doing that, stage1 implementors must pick one:
+
+1. Writing the pid of the container's PID 1 in the file "/var/lib/rkt/pods/$uuid/pid"
+2. Writing the parent pid of the container's PID 1 in the file "/var/lib/rkt/pods/$uuid/ppid". This assumes it has only one child.
```

Comment:
This is a bit confusing. What about

1. Writing to "/var/lib/rkt/pods/$uuid/pid" the PID of the process that is PID 1 in the container
2. Writing to "/var/lib/rkt/pods/$uuid/ppid" the PID of the parent of the process that is PID 1 in the container. This assumes it has only one child.